yt_dlp/extractor/vk.py

```diff
@@ -593,6 +609,17 @@ class VKUserVideosIE(VKBaseIE):
             'id': '-204353299_playlist_426',
         },
         'playlist_mincount': 33,
+    }, {
+    }, {
```

Comment:
this is why the core tests are failing
```suggestion
    }, {
```